tracer/src/Datadog.Trace.Tools.Runner/DumpAnalysis/BCLAssemblyDetector.cs

```diff
@@ -0,0 +1,94 @@
+﻿// <copyright file="BCLAssemblyDetector.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Security.Cryptography;
+
+namespace Datadog.Trace.Tools.Runner.DumpAnalysis
+{
+    internal static class BCLAssemblyDetector
+    {
+        private static readonly string[] MicrosoftPublicKeyTokens =
+        {
+            "b03f5f7f11d50a3a",
+            "b77a5c561934e089",
+            "cc7b13ffcd2ddd51",
+            "31bf3856ad364e35",
+            "7cec85d7bea7798e",
+            "adb9793829ddae60",
+            "30ad4fe6b2a6aeed",
+            "50cebf1cceb9d05e",
+            "979442b78dfc278e",
+            "9dff12846e04bfbd"
```

Comment:
I'm curious about how this list was populated, and how exhaustive we can expect it to be